ledger-api/grpc-definitions/com/daml/ledger/api/v1/ledger_configuration_service.proto

```diff
@@ -50,8 +55,8 @@ message LedgerConfiguration {
   reserved 1; // was min_ttl
   reserved 2; // was max_ttl
 
-  // The maximum value for the ``deduplication_time`` parameter of command submissions
-  // (as described in ``commands.proto``). This defines the maximum time window during which
-  // commands can be deduplicated.
+  // A lower bound on the length of the deduplication periods that can be used for command submissions
+  // (as described in ``commands.proto``). The deduplication period is measured on a local clock
+  // of the participant or Daml ledger.
   google.protobuf.Duration max_deduplication_time = 3;
```

Comment:
@andreaslochbihler-da : I'm unsure about this parameter. I'd have expect that parameter to remain an upper bound on the time-based deduplication options that ledgers MAY enforce. I'll need to do some more digging wrt KV ledgers and their deduplication implementation to understand what we need here. If you already have a good mental model for it, I'd appreciate you sharing it.

Reply:
rename like in write service